reposerver/repository/repository.go

```diff
@@ -1242,17 +1242,17 @@ func (s *Service) GetRevisionMetadata(ctx context.Context, q *apiclient.RepoServ
 	if gpg.IsGPGEnabled() && q.CheckSignature {
 		cs, err := gitClient.VerifyCommitSignature(q.Revision)
 		if err != nil {
-			log.Debugf("Could not verify commit signature: %v", err)
+			log.Errorf("error verifying signature of commit '%s' in repo '%s': %v", q.Revision, q.Repo.Repo, err)
```

Comment:
This is a hard error from `git verify-commit`, and should still be handled as such. I changed log level, and increased verbosity, because it indicates something is wrong.